src/osp/Active/SysNewton.cpp

```diff
@@ -9,33 +9,27 @@ using namespace osp::active;
 
 void cb_force_torque(const NewtonBody* body, dFloat timestep, int threadIndex)
 {
-    SysNewton* sysNewton = static_cast<SysNewton*>(
+    auto* sysNewton = static_cast<SysNewton*>(
                 NewtonWorldGetUserData(NewtonBodyGetWorld(body)));
     ActiveScene &scene = sysNewton->get_scene();
 
-    ACompNwtBody *bodyComp
-            = static_cast<ACompNwtBody*>(NewtonBodyGetUserData(body));
+    auto *bodyComp = static_cast<ACompNwtBody*>(NewtonBodyGetUserData(body));
     DataPhyRigidBody &bodyPhy = bodyComp->m_bodyData;
 
-    //Matrix4 matrix;
+    auto& transform
+            = scene.get_registry().get<ACompTransform>(bodyComp->m_entity);
```

Comment:
Why the new line?

Reply:
There sort of is an 80 column limit rule, but it isn't very well enforced. Perhaps this calls upon more complete coding guidelines.